src/V3OrderCFuncEmitter.h

```diff
@@ -118,8 +118,7 @@ class V3OrderCFuncEmitter final {
             AstNode* const stmtsp = procp->stmtsp();
             UASSERT_OBJ(stmtsp, procp, "Empty process should have been deleted earlier");
             stmtsp->unlinkFrBackWithNext();
-            // Procedure is no longer needed and can be deleted right now
-            VL_DO_DANGLING(procp->deleteTree(), procp);
+            // 'procp' is no longer needed but the OrderGraph still references it (for debug dump)
```

Comment:
Should it be a pushDeletesp instead to avoid leak?

Reply:
Ah, these are unlinked already, good point. It's a bit of a pain to us a deleter here, let me look into it.